modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -55,6 +128,8 @@ bin/neo4j-admin database copy --to-format="block" mydb blockdb
 ----
 CREATE DATABASE blockdb OPTIONS {storeFormat: 'block'}
 ----
+
+[[change-store-format]]
```

Comment:
Should we add the enterprise label to the whole section here? In fact, you can choose or change store format only in the EE.

Reply:
Actually, it is possible to migrate the store format in CE, but there isn't a format to which you can migrate. I guess if you are on standard, you can migrate to aligned. 🤔 I am curious, though, if anyone is using standard.